stargz/stargzify/stargzify.go

```diff
@@ -34,11 +34,10 @@ import (
 )
 
 var (
-	upgrade  = flag.Bool("upgrade", false, "upgrade the image in-place by overwriting the tag")
-	flatten  = flag.Bool("flatten", false, "flatten the image's layers into a single layer")
-	insecAll = flag.Bool("insecure", false, "allow connections to all registries using HTTP")
-	insecSrc = flag.Bool("insecure-src", false, "allow connections to the source registry using HTTP")
-	insecDst = flag.Bool("insecure-dst", false, "allow connections to the destination registry using HTTP")
+	upgrade    = flag.Bool("upgrade", false, "upgrade the image in-place by overwriting the tag")
+	flatten    = flag.Bool("flatten", false, "flatten the image's layers into a single layer")
+	insecure   = flag.Bool("insecure", false, "allow HTTP connections to the registry which has the prefix \"http://\"")
+	httpPrefix = "http://"
```

Comment:
This should be a const, not a var, and it shouldn't be in this var ( ... ) block with the flags.

But really, no need for it to even be a const. Just write it in-line when it's needed once. (if you add a new helper func, it only shows up once.)